debian/control

```diff
@@ -35,13 +35,15 @@ Architecture: any
 Depends: ${misc:Depends},
     clipman,
     dbus,
-    sway,
+    sway-regolith,
     swayidle,
     trawlcat,
     trawld,
     trawldb,
+    gnome-session-bin,
     wl-clipboard,
-    xwayland
+    regolith-look-default
```

Comment:
Hmm, this dependency is going to pull in a lot of X11 stuff I believe.  What specifically is needed here, we may need to break it out into it's own package..

Reply:
Most of the loader script actually remains same for X11 and wayland. I was wondering if it'll be possible for us to have two different binary packages which use different sets of dependencies but installs the same script..

Reply:
I would prefer to keep it simple and not have multiple packages contribute the same file.  I can break out the work in `regolith-look-default`.  Is it just `/usr/share/regolith-look/default_loader.sh` that `regolith-sway` requires or is there more?

Reply:
All I need is default_loader.sh.